AGENTS.md

```diff
@@ -0,0 +1,213 @@
+# AGENTS.md
+
+This file provides context and instructions for AI coding agents working on the **Microsoft Fluent UI Blazor** components library.
+
+## Project Overview
+
+This is a .NET Blazor component library that provides Fluent UI design system components for Blazor applications. The library wraps Microsoft's official **Fluent UI Web Components** and provides additional components leveraging the Fluent Design System.
+
+- **Main Package**: `Microsoft.FluentUI.AspNetCore.Components`
+- **Target Framework**: .NET 8+
+- **License**: MIT
+- **Language**: C# with Razor components
+- **Documentation**: https://www.fluentui-blazor.net
+
+## Repository Structure
+
+```
+src/
+├── Core/                           # Main component library
+├── Core.Scripts/                   # TypeScript/JavaScript for web components
+└── Extensions/                     # Additional packages (EF adapter, OData, etc.)
+
+tests/
+├── Core/                           # Unit tests (bUnit)
+└── Integration/                    # Integration tests
+
+examples/
+├── Demo/                           # Demo application
+└── Samples/                        # Sample projects
+```
+
+## Setup Commands
+
+### Prerequisites
+- .NET 8 SDK or later
+- Node.js 20.15 LTS (for Core.Scripts project)
+- Visual Studio 2022 Preview (recommended on Windows)
+
+### Build Commands
+
+```bash
+# Build the entire solution
+dotnet build Microsoft.FluentUI-v5.sln --configuration Debug
```

Comment:
Why do you specify `--configuration Debug`?

Reply:
My vision of best practise 😃 Removed